### PR TITLE
added an overview

### PR DESCRIPTION
**Pull request by nieuwsma**
### Summary and Scope

Update docs based on a presentation by @rsjostrand-hpe

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+# SLS Overview
+
+## What is currently stored in SLS?
+
+## Hardware information
+
+* Source of truth for HMS Serviers (MEDS, REDS, HMS-Discovery, and HSM) when trying to identify and discovery new hardware in the system
```
Comment (mjendrysik-hpe) [inline docs/overview.md:7]: s/Serviers/Services

File: docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    * MgmtSwitch – River Only
+    * MgmtHLSwitch – River Only
+    * MgmtCDUSwitch – Mountain/Hill
+    * Node
+
+### Network Information
+
+* SLS is the authority for the prescribed layout of management networking on the system.
+* A SLS Network is…
+  * A logical concept. We have allocated a CIDR/IP Address range for a specific purpose. Examples include the
```
Comment (mjendrysik-hpe) [inline docs/overview.md:27]: Move **Examples include the** to a new line and make it
* Examples include:

File: docs/overview.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+### Network Information
+
+* SLS is the authority for the prescribed layout of management networking on the system.
+* A SLS Network is…
+  * A logical concept. We have allocated a CIDR/IP Address range for a specific purpose. Examples include the 
+     * Node Management Network (NMN),
+     * Hardware Management Network (HMN)
+     * Customer Access Network (CAN)
+  * A network is broken into smaller subnets (/22) that fit within the networks CIDR (/17)
+* A SLS Subnet is…
```
Comment (mjendrysik-hpe) [inline docs/overview.md:32]: A SLS Subnet can consist of:
* DHCP Address pool
* etc etc

File: docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+## What services interact with SLS?
+
+* hms-discovery cronjob
+  * River HMN Cabling information to identify unknown hardware
+  * Listing of Management Switches in the System to query via SNMP
+* REDS
+  * Listing of Management NCNs and Columbia Switches
+* MEDS
+  * Looks to SLS for what Mountain/Hill cabinets are in the system
```
Comment (mjendrysik-hpe) [inline docs/overview.md:49]: Really strange sentence for a tech doc. Simplify it, for example:
Queries SLS for Mountain/Hill cabinets

File: docs/overview.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+* Hardware State Manager (HSM)
+  * Role, Subrole, and NID information
+* KEA – DHCP
+  * DHCP Subnet ranges
+* Unbound – DNS
+  * IP Reservations to populate static DNS records. Like packages.local, registry.local, and api-gw-service-nmn.local
```
Comment (mjendrysik-hpe) [inline docs/overview.md:55]: Use an **Examples include:** format.

File: docs/overview.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+  * Node Alias information. For example for the node x3000c0s19b1n0, look to SLS to create a DNS entry for the friendly name automatically ncn-000001-nmn.
```
Comment (mjendrysik-hpe) [inline docs/overview.md:56]: Use an **Examples include:** format.

**Review by mjendrysik-hpe — CHANGES_REQUESTED** [review]

**Review by shunr-hpe — APPROVED** [review]

**Pull request merged**